nat-lab/tests/test_telio_version_compatibility.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+from contextlib import AsyncExitStack
+
+import pytest
+import telio
+from mesh_api import API
+from telio import PathType
+from telio_features import Direct, TelioFeatures
+from utils import ConnectionTag, Ping, new_connection_with_conn_tracker, testing
+from utils.connection_tracker import (
+    ConnectionLimits,
+    generate_connection_tracker_config,
+)
+
+STUN_PROVIDER = ["stun"]
+
+UHP_conn_client_types = [
+    (
+        STUN_PROVIDER,
+        ConnectionTag.DOCKER_CONE_CLIENT_1,
+        ConnectionTag.DOCKER_CONE_CLIENT_2,
+        telio.AdapterType.BoringTun,
+    ),
+]
+
+
+@pytest.mark.asyncio
+@pytest.mark.linux_native
```

Comment:
linux_native mark is used to indicate `LinuxNativeWg`, not that it is only for linux

Reply:
Removed